packages/compass-aggregations/src/components/aggregation-side-panel/stage-wizard-use-cases/match/match-group-form.tsx

```diff
@@ -107,14 +108,22 @@ const nestedGroupContainerStyles = css({
   gap: spacing[3],
 });
 
-const oddNestedGroupStyles = css({
+const oddNestedGroupStylesLight = css({
   background: palette.gray.light3,
 });
 
-const eventNestedGroupStyles = css({
+const eventNestedGroupStylesLight = css({
   background: palette.gray.light2,
 });
 
+const oddNestedGroupStylesDark = css({
+  background: palette.gray.dark3,
+});
+
+const eventNestedGroupStylesDark = css({
+  background: palette.gray.dark2,
+});
```

Comment:
Is `event` a typo?

Reply:
Yes it is thank you @addaleax. I just addressed this in my commit